include/polycap-photon.h

```diff
@@ -45,6 +45,13 @@ struct _polycap_photon;
  */
 typedef struct _polycap_photon                      polycap_photon;
 
+struct _polycap_leak;
```

Comment:
In dit geval moet je de declaratie met de member variables ook hier schrijven, als je will dat deze direct toegankelijk zijn voor gebruikers in C en Python